examples/jit/bernstein_vazirani.ll

```diff
@@ -0,0 +1,536 @@
+; ModuleID = 'Bernstein-Vazirani'
+source_filename = "./bernstein_vazirani.ll"
```

Comment:
Is this file needed or is it an output from PyQIR that could be reproduced without checking the `ll` file in?

Reply:
@troelsfr It is needed in order for the example to work on its own. We didn't want the samples for one project to depend on running another.